### ISSUE TITLE
Movement: Waypoints

### ISSUE THREAD

**Issue by JoelRockes** [integritysystemscompany/animal_schema#66]
Geo-Coordinates appear to only record the start/stop destinations.
Provisions for journey waypoints would make way for enhanced traceability.

**Comment by cookeac** [comment]
Agreed, I have been thinking an Animal Sighted event (optional lat/long or just a location identifier) would allow for both this use case, and also the use case where you just run animals through an EID reader when mustering them for other purposes. It could also be used by the new location-based tags of various sorts.
I'm going to suggest this event is picked up in the next phase of work.
